Lombiq.HelpfulExtensions/Extensions/ShapeTracing/ShapeTracingShapeEvents.cs

```diff
@@ -22,6 +22,15 @@ public Task DisplayedAsync(ShapeDisplayContext context)
         var builder = new HtmlContentBuilder(6);
         var shapeMetadata = context.Shape.Metadata;
 
+        var isPageTitle = shapeMetadata.Type == "PageTitle";
```

Comment:
Replace the hard coded string with `nameof(PageTitleShapes.PageTitle)` (in namespace `OrchardCore.DisplayManagement.Shapes`). It's not likely that the name will even change, but I've seen more egregious breaking changes from OC, so it helps to have a warning.